### PR TITLE
[Yosegi-37] Create POC docker image

### PR DESCRIPTION
**Pull request by koijima**
### What is the necessity of this update? What is updated?
I made the output of logging server yosegi and created a docker image that can be used with the Hive.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+Launch the image with docker-compose.
+
+```
+docker-compose up -d
+```
+
+# Run hive
+
+Ser's results are sent to logger_server from datagen's docker image
```
Comment (yohto) [inline docker/README.md]: I don't understand "Ser's", could you please check spell miss or describe the "Ser's".

Comment (koijima) [reply docker/README.md]: Thank you.
I fixed it.

File: docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/FluentdLogger.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  private FluentLogger logger;
+  private String label;
+  private String messageKey;
+
+  @Override
+  public void setup( final Map setting ) throws IOException {
+    String tag = YamlObjectUtils.getString( setting.get( "tag" ) , "default" );
+    String host = YamlObjectUtils.getString( setting.get( "host" ) , "localhost" );
+    int port = YamlObjectUtils.getInt( setting.get( "port" ) , 24224 );
```
Comment (yohto) [inline docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/FluentdLogger.java]: 24224 is the default of the port of fluentd, is it right?

Comment (koijima) [reply docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/FluentdLogger.java]: Yes, it is the default port for fluentd.

File: docker/hive/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+RUN wget http://ftp.jaist.ac.jp/pub/apache/hadoop/common/hadoop-3.2.0/hadoop-3.2.0.tar.gz
+RUN tar xvf hadoop-3.2.0.tar.gz
+RUN mv hadoop-3.2.0 /opt/hadoop
+RUN rm -rf /opt/hadoop/share/doc
+
+RUN wget http://ftp.jaist.ac.jp/pub/apache/hive/hive-3.1.1/apache-hive-3.1.1-bin.tar.gz
+RUN tar xvf apache-hive-3.1.1-bin.tar.gz
+RUN mv apache-hive-3.1.1-bin /opt/hive
+
+RUN wget http://ftp.yz.yamagata-u.ac.jp/pub/network/apache//db/derby/db-derby-10.14.2.0/db-derby-10.14.2.0-bin.tar.gz
```
Comment (yohto) [inline docker/hive/Dockerfile]: "/"  in the url is duplicated. However, it seems not to modify.

File: docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/Loggers.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  }
+
+  public boolean[] logging( final byte[] message ) throws IOException {
+    boolean[] result = new boolean[loggers.length];
+    for ( int i = 0 ; i < result.length ; i++ ) {
+      result[i] = loggers[i].logging( message );
+    }
+    return result;
+  }
+
```
Comment (yohto) [inline docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/Loggers.java]: I understand the difference between logging2 and logging by cord reading.
However, these names should be described to understand the difference of then, generally.
"2" of logging2 is not mention the difference.
So, please consider to modify the code or pass this version.

**Review by yohto — APPROVED** [review]
I post some comments, but there are not critical.

File: docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/FluentdLogger.java (hunk 2 above)
Comment (yohto) [reply docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/FluentdLogger.java]: Thank you for your consideration. I check the fixation later and approve this PR.

**Comment by yohto** [comment]
@koijima 
This PR seems OK.
Please check my comment and modify the code if you want to.

File: docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/Loggers.java (hunk 4 above)
Comment (koijima) [reply docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/Loggers.java]: sorry.
This method was for verification.
logging2 is deleted.

File: docker/README.md (hunk 1 above)
Comment (yohto) [reply docker/README.md]: Thank you for your consideration. I check the fixation later and approve this PR.

File: docker/hive/Dockerfile (hunk 3 above)
Comment (koijima) [reply docker/hive/Dockerfile]: Thank you.
I fixed it.

**Comment by koijima** [comment]
@yohto 
Thank you for the review.
I fixed it.

File: docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/Loggers.java (hunk 4 above)
Comment (yohto) [reply docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/Loggers.java]: Thank you.

File: docker/hive/Dockerfile (hunk 3 above)
Comment (yohto) [reply docker/hive/Dockerfile]: Thank you.

**Comment by yohto** [comment]
LGTM

**Pull request merged**